pkg/controller/kabaneroplatform/landing.go

```diff
@@ -204,270 +208,239 @@ func getLandingURL(k *kabanerov1alpha1.Kabanero, c client.Client, config *restcl
-		"LANDING_URL/appnav/openshift/appLauncher.js",
-		"LANDING_URL/appnav/openshift/projectNavigation.js",
-	}
+		configMapInstance.BinaryData["kabanero.png"] = b
 
-	ssheets := []string{
-		"LANDING_URL/appnav/openshift/appNavIcon.css",
-	}
+		// TODO: Close response?  cleanup?
 
-	// Replace script URLs with the correct host name.
-	for i, url := range scripts {
-		scripts[i] = strings.Replace(url, "LANDING_URL", landingURL, -1)
+		err = c.Create(context.TODO(), configMapInstance)
+		if err != nil {
+			return err
+		}
 	}
+	
+	
+	// Grab the console object so we can change the icon
+	consoleInstance := &operatorv1.Console{}
+	err = c.Get(context.Background(), types.NamespacedName{
+		Name:      "cluster"}, consoleInstance)
 
-	// Replace stylesheet URLs with the correct host name.
-	for i, url := range ssheets {
-		ssheets[i] = strings.Replace(url, "LANDING_URL", landingURL, -1)
+	// Should be a console instance.... if not, error.
+	if err != nil {
+		return err
 	}
 
-	return scripts, ssheets
-}
+	if consoleInstance.Spec.Customization.CustomLogoFile.Name != configMapName {
+		consoleInstance.Spec.Customization.CustomLogoFile.Name = configMapName
+		consoleInstance.Spec.Customization.CustomLogoFile.Key = "kabanero.png"
 
-// Returns the customization URLs that are not currently defined.
-func getEffectiveCustomizationURLs(extUrls []interface{}, urls []string) []string {
-	var eUrls []string
+		err = c.Update(context.TODO(), consoleInstance)
 
-	for _, url := range urls {
-		if !isInInterfaceList(extUrls, url) {
-			eUrls = append(eUrls, url)
+		if err != nil {
+			return err
 		}
 	}
-	return eUrls
+  */
+	
+	return nil
 }
 
-// Checks if the given string is contained in the given array.
-func isInStringList(urls []string, s string) bool {
-	for _, url := range urls {
-		if url == s {
-			return true
+// Removes customizations from the openshift console.
+func removeWebConsoleCustomization(k *kabanerov1alpha1.Kabanero, c client.Client, reqLogger logr.Logger) error {
+	// Since these are cluster level objects, they cannot set a namespace-level owner and must be
+	// removed manually.
+	consoleLink, err := getConsoleLink(c, "kabanero-app-menu-link")
+	if err == nil {
+		err = c.Delete(context.TODO(), consoleLink)
+		if err != nil {
+			reqLogger.Error(err, "Unable to delete ConsoleLink")
+		}
+	}
+	
+	consoleLink, err = getConsoleLink(c, "kabanero-help-menu-docs")
+	if err == nil {
+		err = c.Delete(context.TODO(), consoleLink)
+		if err != nil {
+			reqLogger.Error(err, "Unable to delete ConsoleLink")
 		}
 	}
-	return false
-}
 
-func isInInterfaceList(urls []interface{}, s string) bool {
-	for _, url := range urls {
-		if url == s {
-			return true
+	consoleLink, err = getConsoleLink(c, "kabanero-help-menu-guides")
+	if err == nil {
+		err = c.Delete(context.TODO(), consoleLink)
+		if err != nil {
+			reqLogger.Error(err, "Unable to delete ConsoleLink")
 		}
 	}
-	return false
+
+	// Remove web console config map and customization.
```

Comment:
I know this code is a no-op, but Kind of confusing to have this code active when the code that adds it is commented out. Might be better to comment it out. The call to get will get executed needlessly

Reply:
yes I agree, I'll comment it out.